docs/source/en/_toctree.yml

```diff
@@ -523,6 +523,8 @@
         title: FocalNet
       - local: model_doc/glpn
         title: GLPN
+      - local: model_doc/grounding-dino
```

Comment:
Grounding DINO can be moved to the multimodal section

Reply:
FIxed here https://github.com/huggingface/transformers/pull/26087/commits/b2fd8687e33e84f12042b4f377c438e47840f14c